Sources/PointFree/Episode/NewShow.swift

```diff
@@ -1,45 +0,0 @@
-import Either
-import HttpPipeline
-import Models
-import PointFreeRouter
-import Prelude
-import Tuple
-import Views
-
-let newEpisodeResponse: M<Tuple4<Either<String, Episode.Id>, User?, SubscriberState, Route?>>
```

Comment:
getting rid of this temporary middleware, and now have it consolidated in the original `episodeResponse` middleware